.github/workflows/build-test.yml

```diff
@@ -553,7 +553,7 @@ jobs:
       # This is the baseline for semver-check. It can be changed,
       # such as if unstable APIs were changed but semver-check
       # didn't know that they were unstable.
-      BASELINE_REF: 2ddba9c1ffc918946c7e7b23f0c5bf6feeb71a3a # a commit on main
+      BASELINE_REF: 5c1235cf4cefbc2a6098e4ad7d098da07c2b5510 # a commit on main
```

Comment:
not a commit on main

you shouldn't need to do this. update the dev version instead